File: example-simple/src/main/java/com/akaita/android/easylauncher/example/MainActivity.java
package com.akaita.android.easylauncher.example;

import android.app.Activity;
import android.app.AlertDialog;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.button).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                new AlertDialog.Builder(MainActivity.this)
                        .setTitle("Hello, Android!")
                        .setMessage("This is an example app.")
                        .setPositiveButton("OK", null)
                        .show();
            }
        });
    }
}

File: example-custom/src/main/java/com/akaita/android/easylauncher/example/MainActivity.kt
package com.akaita.android.easylauncher.example

import android.app.Activity
import android.app.AlertDialog
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        button.setOnClickListener {
            AlertDialog.Builder(this)
                .setTitle("Hello, Android!")
                .setMessage("This is an example app.")
                .setPositiveButton("OK", null)
                .show()
        }
    }
}
